### DISCUSSION TITLE
How can I run the vega branch?

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [requarks/wiki#5368] [Help / Questions]
Hello,

I hope you're well and everything is good for you.

How can I run the vega as been created in the requarks/wiki GitHub repository?

This is my errors:

![1](https://user-images.githubusercontent.com/44679555/172146352-4cb4e819-614f-4bb7-8f16-3a4b2ca6993a.png)
![2](https://user-images.githubusercontent.com/44679555/172146358-2d45353d-6d18-4a88-b283-d05be2a90645.png)

Can somebody share with me the running steps vega branch?

Thank you very much.
Best regards

**Comment by 573077632** [comment]
There seems to be a problem now. I'm up and running, and I can't use V3.0

**Reply by 573077632** [reply]
![(T@5)6{REOV@B1F8ALMQ8}F](https://user-images.githubusercontent.com/102011704/172175201-183dac2f-1704-40c0-be72-36adda1f70ac.png)
![POP1FQ}6_3{UB1_@X(M$697](https://user-images.githubusercontent.com/102011704/172175213-132da71f-ef4a-43d1-9893-a924e559ff00.png)
